kymatio/scattering1d/core/timefrequency_scattering.py

```diff
@@ -281,4 +295,12 @@ def _pad_transpose_fft(coeff_list, total_height, B, fft):
     return out
 
 
+def _transpose_fft(coeff_arr, B, fft):
```

Comment:
I'm not sure that this needs to be its own function. It's only two lines and we don't call it very often.

Reply:
It'll become three with `sc_freq.J_pad[-1]  # TODO`, then we'll remove it entirely once we have `axis=` arg in `fft`